src/modules/Transcript/PriceChart/test.tsx

```diff
@@ -1,63 +1,115 @@
+                        },
+                        {
+                            date: '2021-12-08T22:47:48+00:00',
+                            id: '-6197890410824443545',
+                            price: 641,
+                            priceChangeFromStartPercent: -0.0006,
+                            priceChangeFromStartValue: -0.415,
+                            volume: 1187459,
+                            volumeChangeFromLastPercent: 0.0022,
+                            volumeChangeFromLastValue: 2548,
+                            volumeChangeFromStartPercent: 0.0148,
+                            volumeChangeFromStartValue: 17370,
+                        },
+                        {
+                            date: '2021-12-08T22:48:58+00:00',
+                            id: '-7197890410824443545',
+                            price: 642,
+                            priceChangeFromStartPercent: -0.0006,
+                            priceChangeFromStartValue: -0.415,
+                            volume: 1187459,
+                            volumeChangeFromLastPercent: 0.0022,
+                            volumeChangeFromLastValue: 2548,
+                            volumeChangeFromStartPercent: 0.0148,
+                            volumeChangeFromStartValue: 17370,
+                        },
+                        {
+                            date: '2021-12-08T22:50:58+00:00',
+                            id: '-5197890410824443546',
+                            price: 641,
+                            priceChangeFromStartPercent: -0.0006,
+                            priceChangeFromStartValue: -0.415,
+                            volume: 1187459,
+                            volumeChangeFromLastPercent: 0.0022,
+                            volumeChangeFromLastValue: 2548,
+                            volumeChangeFromStartPercent: 0.0148,
+                            volumeChangeFromStartValue: 17370,
+                        },
+                    ],
+                },
+            ],
+        },
+    ],
+};
 
 describe('PriceChart', () => {
-    test('renders', () => {
+    test('renders', async () => {
         const toggle = jest.fn();
-        const { rendered } = renderWithProvider(
-            <PriceChart headerExpanded={true} priceChartExpanded togglePriceChart={toggle} />
+        const { rendered } = await actAndFlush(() =>
+            renderWithProvider(
+                <PriceChart eventId={'1'} headerExpanded={true} priceChartExpanded togglePriceChart={toggle} />,
+                {
+                    executeQuery: () => fromValue({ data: data }),
+                }
+            )
         );
-        const label = screen.getByText('Price Reaction');
+
+        const label = screen.getByTitle('Chevron');
         fireEvent.click(label);
         expect(rendered.container.querySelector('.highcharts-container')).not.toBeNull();
-        expect(toggle).toHaveBeenCalledTimes(1);
```

Comment:
Don't you still want this since toggle should still be called or is it not for some reason anymore?

Reply:
👍